.pylintrc

```diff
@@ -0,0 +1,551 @@
+# Based on Apache 2.0 licensed code from https://github.com/ClusterHQ/flocker
+
+[MASTER]
```

Comment:
Why do we use exactly this pylint? Is it some best practices? :) Just curious

Reply:
> Wow, so many changes :)
> What do you think about making more PRs with smaller changes (separate PRs for type guessing, pylint, CircleCI)? For now it's fine, but I think it will be much easier to review if we had multiple smaller PRs

hahaha yeah so many changes, mostly because of introducing pylint.
I know reviewing so many changes can be too much, it's my bad. I did put each change in a separate commit to review only the code changes rather than the styling. However, I'll try to make smaller PRs in the future. Thanks for the feedback.